Bottom: 0b3735a814b842937343c06a5cca28cbf1b8a12d
Top:    7bcab7ed6c7de815036a99d397129e39ce2ca095
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 01:24:16 +0100

Remove unused Output.GridGC variable


---

diff --git a/src/global.h b/src/global.h
index f6620c9..bd605ad 100644
--- a/src/global.h
+++ b/src/global.h
@@ -172,8 +172,7 @@ typedef struct			/* holds information about output window */
 {
   hidGC bgGC,			/* background and foreground; */
     fgGC,			/* changed from some routines */
-    pmGC,			/* depth 1 pixmap GC to store clip */
-    GridGC;			/* for the grid */
+    pmGC;			/* depth 1 pixmap GC to store clip */
 
   int Width,			/* sizes of output window (porthole) */
     Height;
diff --git a/src/main.c b/src/main.c
index ab9a614..ad5ee94 100644
--- a/src/main.c
+++ b/src/main.c
@@ -980,7 +980,6 @@ main (int argc, char *argv[])
   Output.bgGC = gui->make_gc ();
   Output.fgGC = gui->make_gc ();
   Output.pmGC = gui->make_gc ();
-  Output.GridGC = gui->make_gc ();
 
   settings_post_process ();
